### PR TITLE
Minor improvements in the `README.md` file

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Minor improvements in the `README.md` file.
  - All the changes are shown in the Git diff. If you have any questions, please let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
-</a>
-<a href="mailto:mrsaadirfan@gmail.com.com">
-  <img align="left" width="26px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/gmail.svg" />
-</a>
-<a href="https://www.youtube.com/channel/UCNQgFgvbz_Vf_2_sWaaKV8Q">
-  <img align="left" width="26px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/youtube.svg" />
-</a>
-
-<br />
+<br/>
```
Comment (msaaddev) [inline README.md]: Let's just remove the line break tag.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+
+<p align="center">
```
Comment (msaaddev) [inline README.md]: Also, let's just remove the center alignment entirely.

File: README.md [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
+
+  <a href="https://twitter.com/msaaddev" target="_blank" rel="noopener noreferrer">
+    <img src="https://img.shields.io/badge/Twitter-msaaddev-blue?logo=twitter&logoColor=blue&color=blue" />
+  </a>
+
+  <a href="mrsaadirfan@gmail.com.com" target="_blank" rel="noopener noreferrer">
+    <img src="https://img.shields.io/badge/Gmail-Saad%20Irfan-red?logo=gmail&logoColor=red&color=red" />
+  </a>
+
+  <br />
```
Comment (msaaddev) [inline README.md]: Also remove this `<br />` so that YouTube badge is also on the same line.

**Review by msaaddev — CHANGES_REQUESTED** [review]

**Comment by msaaddev** [comment]
Thanks again for the PR @Panquesito7! I have requested some changes. Kindly take a look!

**Comment by Panquesito7** [comment]
All fixed. Thanks. 🙂

**Pull request merged**